electron/ipc/handlers.ts

```
@@ -1177,6 +1188,19 @@ function getFfmpegBinaryPath() {
   return ffmpegStatic
 }
 
+function getFfprobeBinaryPath() {
+  const ffprobeStatic = loadFfprobeStatic()
+  if (!ffprobeStatic || typeof ffprobeStatic !== 'string') {
+    throw new Error('FFprobe binary is unavailable. Install @derhuerst/ffprobe-static for this platform.')
+  }
+
+  if (app.isPackaged) {
+    return ffprobeStatic.replace(/\.asar([\/\\])/, '.asar.unpacked$1')
```

Biome (2.4.10)

[error] The character doesn't need to be escaped.

(lint/complexity/noUselessEscapeInRegex)